lib/iris/tests/test_netcdf.py

```diff
@@ -218,6 +218,16 @@ def test_load_merc_grid(self):
         )
         self.assertCML(cube, ("netcdf", "netcdf_merc.cml"))
 
+    def test_load_merc_false_en_grid(self):
```

Comment:
In case no-one has told you before: test modules in the `iris/tests` root is an out-dated way of doing things. Tests should ideally all be under `iris/tests/unit` or `iris/tests/integration` I believe.

But that will make for a very noisy PR so I'm not suggesting we do any refactoring here. Something for its own PR.

Reply:
I assumed so - maybe at some point we should just put them all in the right places?

Reply:
Well, yes, but it would be a couple of days' work and that's time not spent on other things like PyTest or GitHub Actions 🤷